libraries/iio/iio.c

```diff
@@ -47,25 +47,14 @@
 #include "ctype.h"
 #include "tinyiiod.h"
 #include "util.h"
+#include "list.h"
 #include "error.h"
-#include "errno.h"
```

Comment:
we should start using errno. Do not remove it.

Reply:
it was added to error.h